hazelcast-client-new/src/main/resources/hazelcast-client-config-3.5.xsd

```diff
@@ -401,10 +401,10 @@
         </xs:sequence>
     </xs:complexType>
     <xs:complexType name="ssl">
-        <xs:sequence>
+        <xs:all>
```

Comment:
My xsd knowledge is extremely rusty, but can it happen that a user now adds multiple similar sections? E.g. multiple times he configures 'size'.